src/main/java/li/cil/tis3d/common/init/Entities.java

```diff
@@ -18,6 +20,9 @@
 
     static void registerEntities() {
         Registry.register(Registry.ENTITY_TYPE, Constants.NAME_ENTITY_INFRARED_PACKET, INFRARED_PACKET);
+
+        EntityRendererRegistry.INSTANCE.register(INFRARED_PACKET,
```

Comment:
Should probably go to `BootstrapClient`? Crashes dedicated server on startup.

Side question: since it's rendering nothing, I'm guessing it's needed so MC doesn't complain that there's no renderer for the entity?

Reply:
Yes, my bad.

> Side question: since it's rendering nothing, I'm guessing it's needed so MC doesn't complain that there's no renderer for the entity?

Correct, the client aborts at runtime if it doesn't find a registered renderer.

Reply:
OK, cool :)